kconfigs/u-boot_qemu_virt_v7.conf

```diff
@@ -1,5 +1,5 @@
 CONFIG_TEXT_BASE=0x60000000
-CONFIG_BOOTCOMMAND="fdt addr ${fdt_addr} && fdt resize 1000 && smhload zImage ${kernel_addr_r} && smhload rootfs.cpio.gz ${ramdisk_addr_r} ramdisk_addr_end &&  setenv bootargs console=ttyAMA0,115200 earlyprintk=serial,ttyAMA0,115200 && setexpr ramdisk_addr_end ${ramdisk_addr_end} + 1 && fdt chosen ${ramdisk_addr_r} ${ramdisk_addr_end} && bootz ${kernel_addr_r} - ${fdt_addr}"
+CONFIG_BOOTCOMMAND="fdt addr ${fdt_addr} && fdt resize 1000 && load hostfs - ${kernel_addr_r} zImage && load hostfs - ${ramdisk_addr_r} rootfs.cpio.gz && setenv bootargs console=ttyAMA0,115200 earlyprintk=serial,ttyAMA0,115200 && fdt chosen ${ramdisk_addr_r} ${filesize} && bootz ${kernel_addr_r} - ${fdt_addr}"
```

Comment:
When is `${filesize}` set?

Reply:
it will be set set by `load` cmd, in our case we use the value from the latest `load` execution (`load hostfs - ${ramdisk_addr_r} rootfs.cpio.gz`)
from: https://github.com/u-boot/u-boot/blob/v2024.07/doc/usage/semihosting.rst
> The load command will set the filesize variable with the size of the file. The fdt chosen command has been updated to take a size instead of an end address.

Reply:
Got it, thanks.